backend/config.py

```diff
@@ -17,12 +17,14 @@ class DevelopmentConfig(Config):
     TESTING = True
 
     """db config"""
-    SQLALCHEMY_DATABASE_URI = "postgresql://{username}:{password}@{hostname}:{port}/{databasename}".format(
-        username=os.environ.get("DB_USER"),
-        password=os.environ.get("DB_PASSWORD"),
-        hostname=os.environ.get("DB_HOST"),
-        port=5432,
-        databasename=os.environ.get("DB_NAME"),
+    DB_USER = (os.environ.get("DB_USER"),)
+    DB_PASSWORD = (os.environ.get("DB_PASSWORD"),)
+    DB_HOSTNAME = (os.environ.get("DB_HOST"),)
+    DB_PORT = (5432,)
```

Comment:
I don't know much about this project and how it interacts with flask but I'm pretty sure that these variables shouldn't be tuples which they are now. The trick lies in placing comma right after the end of a value.
I'm almost certain that it's not what you want. Consider this example:
```
>>> x = 1,
>>> type(x)
<class 'tuple'>
```